monitoring/values-prom-operator.yaml

```diff
@@ -62,7 +62,7 @@ kubeStateMetrics:
 # https://github.com/helm/charts/tree/master/stable/kube-state-metrics
 kube-state-metrics:
   image:
-    tag: v2.5.0
+    tag: v2.6.0
   resources:
```

Comment:
Hmm, looks like we _did_ update Kube State Metrics...need to include that in the CHANGELOG.

Reply:
Good catch!  I included it in the CHANGELOG for the next push.